rpc/cloudagent/network/networksecuritygroup/moc_cloudagent_networksecuritygroup.proto

```diff
@@ -49,10 +49,9 @@ message NetworkSecurityGroup {
 	string name = 1;
 	string id = 2;
 	repeated NetworkSecurityGroupRule networksecuritygrouprules = 3;
-	string groupName = 5;
-	string locationName = 6;
-	Status status = 7;
-	Tags tags = 8;
+	string locationName = 4;
```

Comment:
I'm assuming no customers are using this. Please let me know if my assumption is wrong.

If any production clusters are using NSG, we should not reuse numbers here.

Reply:
yes, no one is using NSG at the moment!